profile_bluesky/startup/instrument/devices/__init__.py

```diff
@@ -3,9 +3,16 @@
 """
 
 from .aps_source import *
-from .motors import *
 from .scaler import *
 
+from .magnets import *
+from .monochromator import *
+from .huber import *
+from .slits import *
+from .phaseplates import *
+from .others import *
+from .lakeshores import *
+
 # under development
```

Comment:
These items below were placed here when this directory was used for a tutorial.  They could all be removed.  Advice for new installations will discard these anyway.